sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/SQLKeywordSuite.scala

```diff
@@ -30,11 +30,15 @@ import org.apache.spark.sql.catalyst.util.fileToString
 trait SQLKeywordUtils extends SparkFunSuite with SQLHelper {
 
   val sqlSyntaxDefs = {
-    val sqlBasePath = {
+    val sqlBaseParserPath =
```

Comment:
We don't need to read this file now. All the keywords are defined as lexer rules.

Reply:
We still have `ansiNonReserved` `nonReserved` defined in parser file. So the test need to include both files.